Dockerfile

```
@@ -1,13 +1,27 @@
-FROM python:3.12-trixie
+FROM python:3.12-trixie AS builder
+
+COPY pyproject.toml uv.lock dist/imbi_automations*.whl /tmp
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)